examples/kotlin/build.gradle

```diff
@@ -1,10 +1,12 @@
 buildscript {
+    ext.kotlin_version = '1.0.6'
+    
     repositories {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.0.2'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
when updating versions in this sample we should also update the versions in the gradle sample (and if necessary in the maven samples too).